src/shared/user/user.service.js

```diff
@@ -51,15 +51,27 @@ function createUser(body) {
     return User.create(body);
 }
 
+/* istanbul ignore next */
+// Ignored until coverage report will be fixed
+
 /**
  * @function findOrCreateUser
  * @description It will find user by query or create new user in database
  * @param {object} query
  * @param {object} body
  * @return {object}
  */
-function findOrCreateUser (query, body) {
-    return User.findOrCreate(query, body);
+function findOrCreateUser(query, body) {
+    return new Promise((resolve, reject) => {
+        User.findOrCreate(query, body, {}, (err, user) => {
+            if (err) {
```

Comment:
this is redundant, ```User.findOrCreate``` will return promise if you did not pass ```function``` as the last argument